lib/java/desktop-environment/src/main/java/org/enso/desktopenvironment/MacOsDirectories.java

```diff
@@ -4,6 +4,16 @@
 import java.nio.file.Path;
 
 final class MacOsDirectories implements Directories {
+  private static MacOsDirectories instance;
```

Comment:
Just use `new MacOSDirectories()` in https://github.com/enso-org/enso/pull/12618/files#r2018098336

Reply:
Done in b30fa1b8703bcb6ecbdf917e6ad8a2b372b2907f